com.playeveryware.eos/Runtime/EOS_SDK/Generated/Platform/InitializeThreadAffinity.cs

```diff
@@ -39,6 +39,16 @@ public struct InitializeThreadAffinity
 		/// </summary>
 		public ulong RTCIo { get; set; }
 
+		/// <summary>
+		/// Main thread of the external overlay
+		/// </summary>
+		public ulong EmbeddedOverlayMainThread { get; set; }
+
+		/// <summary>
+		/// Worker threads of the external overlay
+		/// </summary>
+		public ulong EmbeddedOverlayWorkerThreads { get; set; }
+
```

Comment:
This will require us to provide settings for these values to be utilized - as well as make certain that they are set within `EOSManager`.

Reply:
I'll create tasks in ClickUp for ingesting these.

Reply:
Err... Well I'll make something. I honestly don't understand what these values are *for*, but I can at least write something up.

Reply:
#EOS-2185 made